app/core/organisation.py

```diff
@@ -60,3 +66,30 @@ def get_organisation_taxonomy_by_name(db: Session, org_name: str) -> TaxonomyDat
         **taxonomy[0],
         "event_types": asdict(entry),
     }
+
+
+def get_organisation_config(db: Session, org: Organisation) -> OrganisationConfig:
+    total = (
+        db.query(FamilyOrganisation)
+        .filter(FamilyOrganisation.organisation_id == org.id)
+        .count()
+    )
+    count_by_category = {}
+    for category in FamilyCategory:
+        count = (
+            db.query(Family)
+            .join(
+                FamilyOrganisation,
+                Family.import_id == FamilyOrganisation.family_import_id,
+            )
+            .filter(FamilyOrganisation.organisation_id == org.id)
+            .filter(Family.family_category == category)
+            .count()
+        )
+        count_by_category[category] = count
```

Comment:
Wonder if this will scale, we might want to look at using a group by instead?

Reply:
done